examples/librispeech/librispeech.ipynb

```diff
@@ -191,6 +187,19 @@
    "cell_type": "code",
    "execution_count": 10,
    "metadata": {},
+   "outputs": [],
+   "source": [
+    "sampler = SingleCutSampler(cutset_dev, shuffle=False, max_samples=16)\n",
```

Comment:
`max_samples=16` is a limit on the audio samples, which is extremely low; I think we might need to find a better name to avoid confusion of "audio samples" with "samples in a batch"...

Reply:
(you probably wanted to specify `max_cuts=16`)

Reply:
but actually, it's more advisable to specify e.g. `max_duration=100.0` (seconds)